code/modules/vehicle/vehicle.dm

```diff
@@ -69,6 +69,11 @@
 		else
 			to_chat(user, "<span class='warning'>[I] seems to be stuck to your hand!</span>")
 		return
+	if(istype(I, /obj/item/borg/upgrade/vtec) && vehicle_move_delay > 1)
```

Comment:
```suggestion
	if(istype(I, /obj/item/borg/upgrade/vtec) && vehicle_move_delay < 1)
```
just a funny little sign error but LGTM otherwise

Reply:
```dm
var/vehicle_move_delay = 2 //tick delay between movements, lower = faster, higher = slower
```

Reply:
Wait its delay so its inversed, im wrong, whoops.

Reply:
yee, appreciate reviewing the changes though <3